### PR TITLE
bulleted points end with : or .

### LINKED ISSUE #491: syntax: review bullet point lists throughout docs
Per https://github.com/iterative/dvc.org/pull/490#discussion_r305564376

Using these guidelines:
- https://www.businesswritingblog.com/business_writing/2005/12/the_best_of_bul.html
- https://www.businesswritingblog.com/business_writing/2012/01/punctuating-bullet-points-.html

### PR DESCRIPTION
**Pull request by Naba7**
Fixes #491

### CODE REVIEW

**Comment by Naba7** [comment]
https://dvc.org/doc/understanding-dvc/collaboration-issues it still has "?" at end of bulleted points. Shall I change or keep it the same. Now, all of the bulleted points end with ":" or "."

File: static/docs/get-started/agenda.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ If you have data files or data sets and/or you produce other data files, models,
...
-  code,
-- track and switch between different versions of the data easily,
+  code.
+- track and switch between different versions of the data easily.
 - being able to answer the question of how data artifacts (e.g. ML models) were
-  built in the first place,
-- being able to compare them,
+  built in the first place.
+- being able to compare them.
 - bring best practices to your team and get everyone on the same page.
```
Comment (jorgeorpinel) [inline static/docs/get-started/agenda.md]: Here, first of all note that you'll need to solve conflicts with my previous commits [648c12f @ agenda.md](https://github.com/iterative/dvc.org/pull/490/commits/648c12f76d3da35866fb55c18e90c5405a2cb685#diff-46e8e4ce4470f73ba360cfdd6e43713e) and [34df9cd @ agenda.md](https://github.com/iterative/dvc.org/pull/490/commits/34df9cd08bab0d34213470b398a2d931e4069f67#diff-46e8e4ce4470f73ba360cfdd6e43713e) (though you can probably just overwrite with your changes after we decide the rules).

However, I think we should rewrite the entire list so it's not a weird sandwich between "and you want to:" and "Then you are in a good place!" (below). I think the last sentence in the previous paragraph can be: "If you have data files or datasets and you produce other data files, datasets, models, etc., then you may want to:". Then we can just remove "Then you are in a good place!" entirely. Also, please change "being" for "be" in the couple bullets where it appears, as "being" is not grammatically correct in this list.

File: static/docs/understanding-dvc/how-it-works.md [hunk 2]
```diff
@@ -90,4 +90,4 @@
    -r--------  2 501  staff   273M Jan 27 03:48 Posts-test.tsv
    ```
 
-8. DVC works on Mac, Linux ,and Windows.
+8. DVC works on Mac, Linux and Windows.
```
Comment (jorgeorpinel) [inline static/docs/understanding-dvc/how-it-works.md]: That comma was correct, just misplaced. Please add it again like this `Linux, and`.

File: static/docs/user-guide/dvc-file-format.md [hunk 3]
```diff
@@ -45,30 +45,30 @@ locked: True
...
-- `outs`: a list of outputs for this stage;
-- `md5`: md5 checksum for this DVC-file;
-- `locked`: whether or not this stage is locked from reproduction;
-- `wdir`: a directory to run command in (default `.`);
+- `cmd`: a command that is being run in this stage.
+- `deps`: a list of dependencies for this stage.
+- `outs`: a list of outputs for this stage.
+- `md5`: md5 checksum for this DVC-file.
+- `locked`: whether or not this stage is locked from reproduction.
+- `wdir`: a directory to run command in (default `.`).
```
Comment (jorgeorpinel) [inline static/docs/user-guide/dvc-file-format.md]: I think that none of the bullets in this doc should have no punctuation at the end (if we agree on the rules I suggested in https://github.com/iterative/dvc.org/issues/491#issuecomment-514118621), as I changed in [34df9cd @ dvc-file-format.md](https://github.com/iterative/dvc.org/pull/490/commits/34df9cd08bab0d34213470b398a2d931e4069f67#diff-a833a0a32123749893e6d04ce58f3834) (pending merge which may take a while so go feel free to apply here again anyway).

File: static/docs/get-started/agenda.md (hunk 1 above)
Comment (Naba7) [reply static/docs/get-started/agenda.md]: alright

**Comment by Naba7** [comment]
The docs containing bulleted points are:
1) changelog
     a) 0.18
     b) 0.35
2) get-started
      a) agenda
      b) configure
      c) example:pipelines
3) user guide
      a) dvc files and directories
      b) file format
      c) contributing
      d) contributing--documentation
      e) shell-autocomplete
      f) usage-analytics
      g) dvcignore
4) tutorial
       a) define-ml-pipeline
5) understanding-dvc
       a) collaboration issues
       b) what-is-dvc
       c) comparison-to-existing-technologies
       d) resources
6) All ``options`` of commands-reference

**Comment by jorgeorpinel** [comment]
Thanks for the comprehensive summary of existing bullet lists @Naba7 ! That will help us decide on the rules... Let me look but for starters I'm pretty sure we can ignore any changelog docs. @shcheklein is that folder even useful? The actual changelog is in the core Github repo releases, I think.

**Comment by shcheklein** [comment]
@jorgeorpinel that folder is useful, I don't usually have time to keep up to date, but the idea is to provide more context for features we are doing. What we can do is to actually create an index.md for the section with a link to the github, so ppl can always find the most recent version.

**Comment by shcheklein** [comment]
@Naba7 hey, so, will you update your PR with this new guide we put together?

**Comment by Naba7** [comment]
I can work on creating an index.md for changelog and link it to GitHub.
@shcheklein Yes, I think I will update it now.
Sorry for delay. I was busy with college work.

**Comment by jorgeorpinel** [comment]
I've extracted the note about `changelog` to #515 so don't worry about it. Please address my review comments above 🙂

**Comment by jorgeorpinel** [comment]
And then you'll also need to resolve conflicts with the master branch. Lmk if you need help with that.

**Comment by Naba7** [comment]
Hi! @jorgeorpinel  I will resolve conflicts with master branch using mergetool. Sorry, I didn't get which two comments you are addressing.

File: static/docs/user-guide/dvc-file-format.md (hunk 3 above)
Comment (Naba7) [reply static/docs/user-guide/dvc-file-format.md]: I have already removed them.

File: static/docs/understanding-dvc/how-it-works.md (hunk 2 above)
Comment (Naba7) [reply static/docs/understanding-dvc/how-it-works.md]: added

**Comment by jorgeorpinel** [comment]
Woops I don't think you resolved any conflict @Naba7 😋 in fact you committed the conflicted files! Please roll back that last commit (for example with `git reset --hard HEAD^`) and try again.

Note that rebasing is preferable to merging but it may require more conflict resolution so up to you. These are relatively advanced Git techniques so again, if you need help please lmk so we coordinate.

Another option you have is to close this PR, but remember [all the changes](https://github.com/iterative/dvc.org/pull/495/files), and then make a new branch off the most current upstream master and apply the changes again from scratch.

**Comment by Naba7** [comment]
Ok, I will try again to resolve conflicts and if I fail I will let you
know. Actually, I want to learn it

**Comment by jorgeorpinel** [comment]
Good job @Naba7 ! Conflicts have been resolved 🙂 just address the rest of my review (listed after this https://github.com/iterative/dvc.org/pull/495#issuecomment-514427232) and lets get this baby merged!

**Comment by Naba7** [comment]
> I've extracted the note about `changelog` to #515 so don't worry about it.

Well yes, I agree. changelog folders isn't that helpful, if it is to be kept then making an index.md and linking to github is a better option.

File: static/docs/get-started/agenda.md [hunk 4]
```diff
@@ -24,7 +24,7 @@ Let the NLP nature of the example not to discourage you from using DVC in other
 Data Science areas. There was no strong reason behind picking the NLP area. On
 contrary, DVC is designed to be pretty agnostic of frameworks, languages, etc.
 If you have data files or data sets and/or you produce other data files, models,
-data sets and you want to:
+datasets, etc., then you may want to:
 
 - Capture and save those <abbr>data artifacts</abbr> the same way we capture
   code
```
Comment (jorgeorpinel) [inline static/docs/get-started/agenda.md]: - [ ] The bullets starting with "Being able" below should use "Be able" instead.

**Comment by jorgeorpinel** [comment]
p.s. @shcheklein in your https://github.com/iterative/dvc.org/pull/495#pullrequestreview-264994410 you just wanted us to set rules which we have done in https://github.com/iterative/dvc.org/issues/491#issuecomment-514118621 so you could approve the "changes" any time.

**Comment by jorgeorpinel** [comment]
@Naba7 another thing missing here is to review all the docs. I see many files that still have bullets ending in `;`, for example. Search files for regex `^\-.*\n?.*;`:

image

File: static/docs/understanding-dvc/related-technologies.md [hunk 5]
```diff
@@ -9,9 +9,9 @@ process.
 
 1. **Git**. The difference is:
 
-   - DVC extends Git by introducing the concept of _data files_ - large files
-     that should NOT be stored in a Git repository but still need to be tracked
-     and versioned.
+   - DVC extends Git by introducing the concept of _data files_ which are large
+     files that should NOT be stored in a Git repository but still needs to be
```
[the remark this answers is not in the document]
Comment (Naba7) [reply static/docs/understanding-dvc/related-technologies.md]: ``--``  ``-``

File: static/docs/understanding-dvc/related-technologies.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ process.
    - DVC doesn't need to run any services. No graphical user interface as a
      result, but we expect some GUI services will be created on top of DVC.
 
-   - DVC has transparent design:
+   - DVC has transparent design which
```
[the remark this answers is not in the document]
Comment (Naba7) [reply static/docs/understanding-dvc/related-technologies.md]: I used ``of`` instead and used a comma before ``have a human-readable...``
```
DVC has transparent design of meta files and directories(including the data cache), have a human-readable format and can be easily reused by external tools.
```

File: static/docs/user-guide/dvc-files-and-directories.md [hunk 7]
```diff
@@ -1,16 +1,12 @@
...
-  hand or with a special command: `dvc config`.
+### Special DVC internal files and directories
 
-- `.dvc/config.local` - this is a local configuration file, that will overwrite
-  options in `.dvc/config`. This is useful when you need to specify private
-  options in your config that you don't want to track and share through Git
-  (credentials, private locations, etc). The local config file can be edited by
-  hand or with a special command: `dvc config --local`.
+`.dvc/config` - this is a configuration file. The config file can be edited by
+hand or with a special command: `dvc config`.
```
[the remark this answers is not in the document]
Comment (Naba7) [reply static/docs/user-guide/dvc-files-and-directories.md]: My bad! I restored those.

**Comment by Naba7** [comment]
@jorgeorpinel I think only ``commands-reference`` module only have ``;`` ending. Working on that.

File: static/docs/understanding-dvc/related-technologies.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply static/docs/understanding-dvc/related-technologies.md]: Hmmm you changed it to
```
   - DVC extends Git by introducing the concept of _data files_ which are large
     –files that should NOT be stored in a Git repository but still need to be
```
It should be
```
   - DVC extends Git by introducing the concept of _data files_ – large files
     that should NOT be stored in a Git repository but still need to be
```
(check 80 char wrapping)

File: static/docs/understanding-dvc/related-technologies.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply static/docs/understanding-dvc/related-technologies.md]: Yeah I see that thanks... But that paragraph doesn't make sense now. Let's just restore it the way it was with `:` please.

File: static/docs/commands-reference/pull.md [hunk 8]
```diff
@@ -200,4 +200,3 @@ the `model.p.dvc` stage occurs later, its data was not pulled.
 Then we ran `dvc pull` specifying the last stage, `model.p.dvc`, and its data
 was downloaded. Finally, we ran `dvc pull` with no options to make sure that all
 data was already pulled with the previous commands.
-
```
Comment (jorgeorpinel) [inline static/docs/commands-reference/pull.md]: Please don't remove line breaks at the end of files. If this is your IDE or code editor please configure it to not do this.

File: static/docs/commands-reference/push.md [hunk 9]
```diff
@@ -339,4 +339,3 @@ Pipelines are up to date. Nothing to reproduce.
 
 And running `dvc status --cloud` verifies that indeed there are no more files to
 upload to the remote cache.
-
```
Comment (jorgeorpinel) [inline static/docs/commands-reference/push.md]: Same. Don't remove line breaks at the end of files please.

File: static/docs/commands-reference/unprotect.md [hunk 10]
```diff
@@ -29,10 +29,10 @@ on this process.
 
 `dvc unprotect` can be an expensive operation (involves copying data), check
 first whether your task matches one of the cases that are considered safe, even
-when cache protected mode is enabled:
+when cache protected mode is enabled by:
 
-- Adding more files to a directory input data set (say, images or videos)
-- Deleting files from a directory data set
+- Adding more files to a directory input data set (say, images or videos).
+- Deleting files from a directory data set.
```
Comment (jorgeorpinel) [inline static/docs/commands-reference/unprotect.md]: This is not a full sentence so no `.`. Please undo.

**Comment by Naba7** [comment]
hi @jorgeorpinel I have changed and updated all bullet points. I think its ready to be merged.

**Comment by shcheklein** [comment]
@Naba7 thanks! looks amazing.

@jorgeorpinel please take a look and let me know if you see any other issues there

File: static/docs/understanding-dvc/related-technologies.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply static/docs/understanding-dvc/related-technologies.md]: This is not resolved yet. I posted the exact text to use in the comment above. Please apply.

File: static/docs/user-guide/contributing-documentation.md [hunk 11]
```diff
@@ -12,10 +12,10 @@ To contribute documentation you need to know these locations:
 - [Content](https://github.com/iterative/dvc.org/tree/master/static/docs)
   (`/static/docs`) -
   [Markdown](https://guides.github.com/features/mastering-markdown/) files of
-  the different pages to render dynamically in the browser;
+  the different pages to render dynamically in the browser
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing-documentation.md]: This bullet should be a sentence so the list is consistent (the other 2 bullets are sentences and end in period `.`). Let's also switch to `:` instead of `-` here. Please use:

```
- [Content](https://github.com/iterative/dvc.org/tree/master/static/docs)
  (`/static/docs`): Place
  [Markdown](https://guides.github.com/features/mastering-markdown/) files of
  the different pages to render dynamically in the browser.
```

File: static/docs/user-guide/contributing-documentation.md [hunk 12]
```diff
@@ -12,10 +12,10 @@ To contribute documentation you need to know these locations:
...
 - [Images](https://github.com/iterative/dvc.org/tree/master/static/img)
   (`/static/img`) - add new images, gif, svg, etc here. Reference them from the
-  Markdown files like this: `![](/static/img/reproducibility.png)`;
+  Markdown files like this: `![](/static/img/reproducibility.png)`
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing-documentation.md]: Please change `) - add` for `): Add` in the beginning of this bullet, and end it with period `.`.

File: static/docs/user-guide/contributing-documentation.md [hunk 13]
```diff
@@ -12,10 +12,10 @@ To contribute documentation you need to know these locations:
...
 - [Sections](https://github.com/iterative/dvc.org/tree/master/src/Documentation/sidebar.json)
   (`.../sidebar.json`) - edit it to register a new section for the navigation
   menu.
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing-documentation.md]: Let's change `) - edit` for `): Edit` please.

File: static/docs/user-guide/contributing.md [hunk 14]
```diff
@@ -288,11 +288,11 @@ Fixes #(Github issue id).
 Message types:
 
 - *component* - name of a component that this patch is affecting. Use `dvc` in a
-  general case;
-- _short description_ - short description of the patch;
+  general case.
+- _short description_ - short description of the patch.
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing.md]: No period `.` here please. Not a full sentence.

File: static/docs/user-guide/contributing.md [hunk 15]
```diff
@@ -288,11 +288,11 @@ Fixes #(Github issue id).
...
 - _long description_ - If needed, longer message describing the patch in more
-  details;
-- _github issue id_ - An id of the Github issue that this patch is addressing
+  details.
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing.md]: Start bullet with lower case (`if needed`), no period `.` at the end.

File: static/docs/user-guide/contributing.md [hunk 16]
```diff
@@ -288,11 +288,11 @@ Fixes #(Github issue id).
...
+- _github issue id_ - An id of the Github issue that this patch is addressing.
```
Comment (jorgeorpinel) [inline static/docs/user-guide/contributing.md]: Start bullet this (lower case) text: `id of the Github...` (just remove `An`). No period . at the end.

File: static/docs/user-guide/dvc-files-and-directories.md [hunk 17]
```diff
@@ -1,7 +1,9 @@
 # DVC Files and Directories
 
 Once initialized in a project, DVC populates its installation directory
-(`.dvc/`) with special DVC internal files and directories:
+(`.dvc/`) with special DVC internal files and directories.
+
+### Special DVC internal files and directories
```
Comment (jorgeorpinel) [inline static/docs/user-guide/dvc-files-and-directories.md]: Unnecessary change, please undo. But thanks for the initiative!

File: static/docs/understanding-dvc/related-technologies.md [hunk 18]
```diff
@@ -9,9 +9,9 @@ process.
 
 1. **Git**. The difference is:
 
-   - DVC extends Git by introducing the concept of _data files_ - large files
-     that should NOT be stored in a Git repository but still need to be tracked
-     and versioned.
+   - DVC extends Git by introducing the concept of _data files_ which are large
+     –files that should NOT be stored in a Git repository but still need to be
+     tracked and versioned.
```
[the remark this answers is not in the document]
Comment (Naba7) [reply static/docs/understanding-dvc/related-technologies.md]: sure

**Comment by Naba7** [comment]
Thanks @jorgeorpinel  Yes, my bad! I have changed those. Please take a look!

File: static/docs/user-guide/dvc-files-and-directories.md [hunk 19]
```diff
@@ -36,9 +38,9 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/updater` - this file is used store latest available version of dvc,
   which is used to remind user to upgrade.
 
-- `.dvc/updater.lock` - a lock file for `.dvc/updater`.
+- `.dvc/updater.lock` - lock file for `.dvc/updater`
 
-- `.dvc/lock` - a lock file for the whole dvc project.
+- `.dvc/lock` - lock file for the whole dvc project
```
Comment (jorgeorpinel) [inline static/docs/user-guide/dvc-files-and-directories.md]: Thanks! I think we should change these to `:` instead of `-` and start in capital letter for consistency, now that I think about it. But I'll do it myself, don't worry about it.

File: static/docs/understanding-dvc/related-technologies.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply static/docs/understanding-dvc/related-technologies.md]: thanks
